prereq_tools/base.py

```diff
@@ -530,6 +533,10 @@ def download(self, basename):
 
             print "Try #%d to get %s failed due to md5 mismatch" % (i + 1,
                                                                     self.url)
+            if (i + 1) != tries:
```

Comment:
Don't we want this sleep to run in the line 526 failure case also?

Reply:
good point